quartz/components/scripts/search.inline.ts

```diff
@@ -164,12 +268,183 @@ async function setupSearch(searchElement: Element, currentSlug: FullSlug, data:
   const searchLayout = searchElement.querySelector(".search-layout") as HTMLElement
   if (!searchLayout) return
 
+  const searchSpace = searchElement?.querySelector(".search-space") as HTMLFormElement
+  if (!searchSpace) return
+
+  // Create semantic search progress bar
+  const progressBar = document.createElement("div")
+  progressBar.className = "semantic-search-progress"
+  progressBar.style.cssText = `
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    height: 2px;
+    width: 0;
+    background: var(--secondary);
+    transition: width 0.3s ease, opacity 0.3s ease;
+    opacity: 0;
+    z-index: 9999;
```

Comment:
Can these styles be moved to the stylesheet and attached to the `semanti-search-progress`?